docs/sphinx_doc/dev/verification.rst

```diff
@@ -0,0 +1,83 @@
+.. _dev-verification:
+
+Verification
+============
+
+Verification of AMR-WIND uses `MASA <https://github.com/manufactured-solutions/MASA>`_ and auto-differention tools to implement the Method of Manufactured Solutions.
+
+Building and linking MASA
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The user must first build and install `MASA
+<https://github.com/manufactured-solutions/MASA>`_. This can be done
+from source or using `Spack <https://spack.io>`_.
+
+Building MASA from source
+#########################
+
+The user must build both `Metaphysicl
+<https://github.com/roystgnr/MetaPhysicL>`_ and MASA. After defining
+``METAPHYSICL_ROOT_DIR`` and ``MASA_ROOT_DIR``:
+
+.. code-block:: bash
+
+   $ git clone https://github.com/roystgnr/MetaPhysicL
+   $ ./bootstrap
+   $ ./configure --prefix=$METAPHYSICL_ROOT_DIR
+   $ make
+   $ make install
+
+.. code-block:: bash
+
+   $ git clone https://github.com/manufactured-solutions/MASA
+   $ ./bootstrap
+   $ ./configure --enable-fortran-interfaces METAPHYSICL_DIR=$METAPHYSICL_ROOT_DIR --prefix=$MASA_ROOT_DIR --enable-python-interfaces
+   $ make
+   $ make check
+   $ make install
+
+Building MASA using Spack
+#########################
+
+Assuming the user has Spack configured for their system, building and
+installing MASA is as easy as:
+
+.. code-block:: bash
+
+   $ spack install masa
+
+Linking MASA to AMR-Wind
+########################
+
+The following CMake options enable MASA in AMR-Wind:
+
+.. code-block:: bash
+
+   -DAMR_WIND_ENABLE_VERIFICATION=ON -DAMR_WIND_ENABLE_MASA=ON -DMASA_DIR=$MASA_DIR
```

Comment:
it might be less confusing if you use MASA_ROOT_DIR like above. Or vice-versa so they are consistent 

mkdir build
cd build
cmake -DAMR_WIND_ENABLE_VERIFICATION=ON -DAMR_WIND_ENABLE_MASA=ON -DMASA_DIR=$MASA_ROOT_DIR ..

Reply:
good idea, let me do that